css/ampdoc.css

```diff
@@ -774,61 +293,14 @@ amp-experiment, amp-share-tracking {
   padding: 5px;
 }
 
-/**
- * amp-accordion to avoid FOUC.
- */
-
-/* Non-overridable properties */
-amp-accordion {
-  display: block !important;
-}
-
-/* Make sections non-floatable */
-amp-accordion > section {
-  float: none !important;
-}
-
-/*  Display the first 2 elements (heading and content) */
-amp-accordion > section > * {
-  float: none !important;
-  display: block !important;
-  overflow: hidden !important; /* clearfix */
-  position: relative !important;
-}
-
-amp-accordion,
-amp-accordion > section,
-.i-amphtml-accordion-header,
```

Comment:
`i-amphtml-accordion-header` just shouldn't be here. period.

Reply:
all the amp-accordion related css are moved to the shared file, as they are needed by ads